jwst/cube_build/cube_build.py

```diff
@@ -874,10 +872,10 @@ def setup_IFUCube(self,j):
 
         IFUCube.meta.wcsinfo.wcsaxes = 3
 
-        IFUCube.flux_extension = 'SCI'
-        IFUCube.error_extension = 'ERR'
+        IFUCube.meta.flux_extension = 'SCI'
+        IFUCube.meta.error_extension = 'ERR'
         IFUCube.meta.dq_extension = 'DQ'
-        IFUCube.meta.weightmap = 'WMAP'
+        IFUCube.meta.weightmap_extension = 'WMAP'
```

Comment:
The IFUCubeModel data model schema doesn't have a "weightmap_extension" attribute defined. The only ones that David Law asked us to define, to keep in sync with other IFU projects, were flux, error, and dq. So you don't need line 878.